### PR TITLE
ENH: Add full content

### PR DESCRIPTION
**Pull request by sappelhoff**
Today I did a first pass at our supplementary material to be covered with the webpage. 

@tstenner let me know if you find something worth changing.

For larger things to change, I would prefer to first merge this PR and then iterate towards the "perfect" solution.

I'd rather have a published draft than a perfect document in my file drawer :-)

---

# Preview

https://23-253906639-gh.circle-artifacts.com/0/html/index.html

### CODE REVIEW

File: docs/diy-instructions.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+Firmware
+--------
+
+This is the firmware that should be uploaded to the microcontroller.
+Take care to define the ``outputPins`` according to how you connected your wires.
+
+.. code-block:: C
+
+   #include "Arduino.h";
```
Comment (tstenner) [inline docs/diy-instructions.rst]: This should be included from a file so it's always up to date

File: docs/diy-instructions.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+   // the following pin numbers depend on where you actually put the wires on your device
+   const int outputPins[] = {2,3,4,5,6,7,8,9};
```
Comment (tstenner) [inline docs/diy-instructions.rst]: Are those the default pins for the Leonardo?

File: docs/commercial-alternatives.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| Cedrus C-pod                  | `Cedrus Corporation`_        | PIC32MX150F128                                      | https://cedrus.com/c-pod/                                         |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| CORTECH SOLUTIONS SD-MS-TCPUA | `Cortech Solutions Inc`_     | National Instruments digital IO device, NI USB 6501 | https://cortechsolutions.com/product/sd-ms-tcpua/                 |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| USB2TTL8 Adapter              | `LabHackers Inc`_            | MK20DX256                                           | http://labhackers.com/usb2ttl8.html                               |
```
Comment (tstenner) [inline docs/commercial-alternatives.rst:31]: Link to PJRC / Teensy?

File: docs/code-examples.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   # Set data status to `0`
+   ser.write(bytes([0]))
+
+   # Let's send some information: Bytes from 1 to 10
+   # We assume that our recording device has a sampling rate of 1000Hz
+   for data in range(1, 10):
+       byte_to_send = bytes([data])
+
+       ser.write(byte_to_send)  # send data
+       mysleep(waitsecs=0.001)  # wait for 1ms for recording device to catch this
```
Comment (tstenner) [inline docs/code-examples.rst]: Not needed with the delay and auto-reset in the firmware

**Review by tstenner — CHANGES_REQUESTED** [review]

File: docs/commercial-alternatives.rst (hunk 3 above)
Comment (sappelhoff) [reply docs/commercial-alternatives.rst:31]: in that list I'd only refer to "finished products" ... without assembly or anything else --> "commercial alternatives", not "list of viable microcontrollers"

File: docs/code-examples.rst (hunk 4 above)
Comment (sappelhoff) [reply docs/code-examples.rst]: true, thanks - fixed it!

File: docs/diy-instructions.rst (hunk 2 above)
Comment (sappelhoff) [reply docs/diy-instructions.rst]: These are the first available digital pins on the Leonardo --> it's the way I set up my prototype

File: docs/diy-instructions.rst (hunk 1 above)
Comment (sappelhoff) [reply docs/diy-instructions.rst]: I agree, thanks and fixed.

**Pull request merged**